### PR TITLE
Fixes to 2021 EPA + small issues

### PR DESCRIPTION
**Pull request by patstirling**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Waste, mostly

**What?**

Fixes these issues:

- number residual oil and distillate oil EFs correctly
- Anthracite coal remove /1000 on the EF
- Waste EFs *1000 to convert to kg rather than metric tons CO2e
- remove a bunch of "
- fix some incorrect contribution dates

**Why?**

Data incorrect/wrongly formatted

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -4968,81 +4968,81 @@ Waste,Electrical Waste,waste_type_weee_mixed-disposal_method_open_loop,WEEE wast
...
+Waste,Food and Organic Waste,waste_type_mixed_organics-disposal_method_combusted_aviation,Mixed Organics - Combusted,short ton,not-supplied,50,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,EPA,2021,2021,2020,US,,Climatiq,2022/01/22,Emission intensity of disposing of the specified material through cumbustion. These factors do not include any avoided emissions impact from any of the disposal methods. Combustion emissions include transport to combustion facility and combustion-related non-biogenic CO2 and N2O. Retrieved from the EPA's GHG Emission Factors Hub (xlsx).,https://www.epa.gov/climateleadership/ghg-emission-factors-hub
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:5028]: Do we need "aviation" in the name as well?

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -5097,41 +5097,41 @@ Waste,Metal Waste,waste_type_metal_aluminium_cans_and_foil_excluding_forming-dis
...
+Waste,Paper and Cardboard Waste,waste_type_mixed_paper_primarily_from_offices-disposal_method_combusted_aviation,Mixed Paper (primarily from offices) - Combusted,short ton,not-supplied,50,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,EPA,2021,2021,2020,US,,Climatiq,2022/01/22,Emission intensity of disposing of the specified material through cumbustion. These factors do not include any avoided emissions impact from any of the disposal methods. Combustion emissions include transport to combustion facility and combustion-related non-biogenic CO2 and N2O. Retrieved from the EPA's GHG Emission Factors Hub (xlsx).,https://www.epa.gov/climateleadership/ghg-emission-factors-hub
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:5123]: Same comment.

**Review by maryarbab — APPROVED** [review]

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:5123]: fixed!

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:5028]: fixed! Thanks

**Review by maryarbab — APPROVED** [review]

**Pull request merged**